src/plugins/service-worker/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+const buildPaths = require('mwp-cli/src/commands/buildCommands/config/paths');
```

Comment:
I do have a recommendation and I should make a ticket as a follow up -

we need a method of providing project level config. babel and jest have ways of doing it (package.json content or separate config files) so it should be possible with our own modules.